src/Database/Concerns/HasRelationships.php

```diff
@@ -416,17 +429,17 @@ public function belongsTo($related, $foreignKey = null, $parentKey = null, $rela
      * Overridden from {@link Eloquent\Model} to allow the usage of the intermediary methods to handle the relation.
      * @return \October\Rain\Database\Relations\BelongsTo
      */
-    public function morphTo($name = null, $type = null, $id = null)
+    public function morphTo($name = null, $type = null, $id = null, $ownerKey = null)
     {
         if (is_null($name)) {
             $name = $this->getRelationCaller();
         }
 
-        list($type, $id) = $this->getMorphs(Str::snake($name), $type, $id);
+        [$type, $id] = $this->getMorphs(Str::snake($name), $type, $id);
```

Comment:
@bennothommo what's the reason for this change?

Reply:
@LukeTowers Is "habit" an acceptable answer? :P

Reply:
@bennothommo only if you can back it up with a reason 😛

Reply:
@LukeTowers It's just the more "modern" way of defining a `list()` call. I probably did it when I went through that class, but I can change it back. It's purely cosmetic.

Reply:
I think I'd prefer the list call, at least with that if I don't understand what's happening I can google the function name and figure out how the list() magic works.